bin/NativeTests/CodexTests.cpp

```diff
@@ -0,0 +1,215 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#include "stdafx.h"
+#include "catch.hpp"
+#include <process.h>
```

Comment:
Yeah :( I want these tests to run on xplat too- @akroshg, is that on your radar?